### PR TITLE
[feat] - add cronjob for creating aws snapshots

### PR DESCRIPTION
**Pull request by mur-me**
Hi @witnsby,

Please take a look on the PR, especially on the modules variable section.

What was done:
* added snapshots logic to the module `cronjob_aws_snapshots_managment`
* added volume_snapshot_class as separate thing, 'cause it is creating one time for cluster
* rewritten deleter logic - now it is bash script with logs
* added watch verb for role, because I've faced `vendor/k8s.io/client-go/tools/watch/informerwatcher.go:146: Failed to watch *unstructured.Unstructured: unknown` error

Eager for your review! 

**Note: right now it is running only against 1G test pvc, I want to discuss how many snapshots we will keep and how often we will do them. 
After this I'll enabled volume snapshots for all needed pv claims in dev/mainnet.**

### CODE REVIEW

File: k8s/modules_aws_snapshots_dev.tf [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+#   volume_name = "calibrationapi-archive-node"
+#   namespace   = kubernetes_namespace_v1.network.metadata[0].name
+# }
+
+# module "wallaby_archive_snapshot" {
+#   count       = local.is_dev_envs
+#   source      = "../modules/cronjob_aws_snapshots_managment"
+#   volume_name = "wallaby-archive"
+#   namespace   = kubernetes_namespace_v1.network.metadata[0].name
+# }
```
Comment (witnsby) [inline k8s/modules_aws_snapshots_dev.tf:23]: could you configure your IDE to automatically add new line in the end of the file

File: k8s/modules_aws_snapshots_dev.tf [hunk 2]
```diff
@@ -0,0 +1,23 @@
+module "test-calibrationapi" {
```
Comment (witnsby) [inline k8s/modules_aws_snapshots_dev.tf:1]: do we need to have test word in the module name?

**Review by witnsby — CHANGES_REQUESTED** [review]

File: k8s/modules_aws_snapshots_dev.tf (hunk 1 above)
Comment (mur-me) [reply k8s/modules_aws_snapshots_dev.tf:23]: Thanks for suggestion, done it :smile:

File: k8s/modules_aws_snapshots_dev.tf (hunk 2 above)
Comment (mur-me) [reply k8s/modules_aws_snapshots_dev.tf:1]: 1. Right now I've added only testing stuff here, I want to discuss how often we should do them and how many to keep, especially in mainnet.
2. After discussion I'll enable them for dev cluster node, check that everything is fine
3. Enable in mainnet

**Comment by mur-me** [comment]
Added mainnet job

**Comment by mur-me** [comment]
I haven't apply this to prod yet because to redeployment of `aws_api_gateway_deployment` for some reason

**Comment by mur-me** [comment]
Applied to the mainnet - @ArseniiPetrovich, please review and approve it, thanks!

**Pull request merged**